src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java

```diff
@@ -276,6 +259,23 @@ public final void log(int lineNo, int colNo, String key,
                 getCustomMessages().get(key)));
     }
 
+    /**
+     * Returns the lines associated with the tree.
+     * @return the file contents
+     */
+    public final String[] getLines() {
+        return context.get().fileContents.getLines();
+    }
+
+    /**
+     * Returns the line associated with the tree.
+     * @param index index of the line
+     * @return the line from the file contents
+     */
+    public final String getLine(int index) {
```

Comment:
Why you moved this method here, I see no usage of it from other class methods 
Moving this method doesn't affect penalty score, am I wrong?

Reply:
No particular reason, I just wanted to keep it close to the `getLines()` method, since they've got similar identifiers.